test/gpu/native/studies/shoc/sort.gpu-keys

```diff
@@ -0,0 +1,3 @@
+Sort Rate  25165824items Median:
```

Comment:
this looks too typo-y, is it really how we (and reference?) report the result?

Reply:
I've updated to include a space.